src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -10185,7 +10688,25 @@ namespace HVACVariableRefrigerantFlow {
 
         // Otherwise the coil needs to turn on. Get full load result
         PartLoadRatio = 1.0;
-        this->CalcVRF_FluidTCtrl(VRFTUNum, FirstHVACIteration, PartLoadRatio, FullOutput, OnOffAirFlowRatio);
+        this->CalcVRF_FluidTCtrl(VRFTUNum, FirstHVACIteration, PartLoadRatio, FullOutput, OnOffAirFlowRatio, SuppHeatCoilLoad);
```

Comment:
This call looks correct, using this->Calc. Compare to 8265 above.

Reply:
The reason he couldn't use `this->` above is because he wasn't in a member function there, so there's no `this->` reference.  `this->` is only available when you are already in the context of a single instance.